frontend/src/app/main/data/workspace/svg_upload.cljs

```diff
@@ -73,34 +73,34 @@
                     :else (str tag))))
 
 (defn setup-fill [shape]
-  (cond-> shape
+  (let [color-attr (str/trim (get-in shape [:svg-attrs :fill]))
+        color-attr (if (= color-attr "currentColor") "#000000" color-attr)
```

Comment:
I think it would be better using clr/black

Reply:
done